src/screens/OtherProfile/index.js

```diff
@@ -305,7 +305,7 @@ const OtherProfile = () => {
             <CircleGradient
               fill={dataMain.karma_score ?? 0}
               size={normalize(100)}
-              width={normalize(2)}>
+              width={normalize(3)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The width of the CircleGradient component has been increased from `normalize(2)` to `normalize(3)`. Ensure that this change doesn't distort the visual appearance of the component, especially on different screen sizes and resolutions.
```diff
- width={normalize(2)}
+ width={normalize(3)}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->